gen3.datastage.io/portal/gitops.json

```diff
@@ -209,11 +202,44 @@
             "bmi",
             "age_at_index"
           ]
+        },
+        {
+          "title": "Diagnosis",
+          "fields": [
+            "diastolic_blood_pressure",
+            "systolic_blood_pressure"
+          ]
         }
       ]
     },
     "table": {
-      "enabled": true
+      "enabled": true,
+      "fields": [
```

Comment:
I assume this is something that windmill now uses to know which fields to show? and is this compatible with the version of windmill currently in STAGE?

Reply:
yup! If you don't provide all the fields Guppy shows them all by default
but its weird because things like auth resource path show up so we are specifying them